### DISCUSSION TITLE
Temperature sea water

### DISCUSSION THREAD

**Discussion by kpkingg** [open-meteo/open-meteo#814] [Q&A]
Hello,

I'm looking for data about Tempearture of the sea water, there that data in the open-meteo API ?

**Comment by patrick-zippenfenig** [comment]
Partly it is possible. You can use soil temperature on 0cm or surface temperature and set cell_selection=sea: https://api.open-meteo.com/v1/forecast?latitude=54.3213&longitude=10.1349&hourly=soil_temperature_0cm&cell_selection=sea

**Reply by meb427** [reply]
Not sure it can work that way. Check the lakes, the inertia of water temperature is quite different. While soil can go to 0c, the lake will remain hotter and same when it gets hotter, it take a lot of time.  Same for the sea.
And checking the link you posted, I see the water temperature is from moving 8 to 18 during the day. Which sound not right.

**Reply by patrick-zippenfenig** [reply]
Hm... I will have to check again the definition. It used to work this way. I will try to include sea surface temperature into the marine API with the next update.
